src/cascadia/TerminalSettingsEditor/ActionsViewModel.cpp

```diff
@@ -299,10 +295,15 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         if (newName.empty())
         {
             // if the name was cleared, refresh the DisplayName
-            _NotifyChanges(L"DisplayName");
+            _NotifyChanges(L"DisplayName", L"DisplayNameAndKeyChordAutomationPropName");
         }
     }
 
+    winrt::hstring CommandViewModel::DisplayNameAndKeyChordAutomationPropName()
+    {
+        return DisplayName() + L" " + FirstKeyChordText();

```

Comment:
```suggestion
        return DisplayName() + L", " + FirstKeyChordText();
```
The comma makes the screen reader read it much more nicely